### PR TITLE
[NUI] Fix CA2000 issues for Render Shadow

### PR DESCRIPTION
**Pull request by zhouleonlei**
### Description of Change ###
Fix CA2000 issues for Render Shadow

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by elishateng — APPROVED** [review]
LGTM

**Review by AchoWang — APPROVED** [review]
Looks good to me.

File: src/Tizen.NUI/src/public/Renderer.cs [hunk 1]
```diff
@@ -47,12 +47,16 @@ public int DepthIndex
             get
             {
                 int temp = 0;
-                Tizen.NUI.Object.GetProperty(SwigCPtr, Renderer.Property.DepthIndex).Get(out temp);
+                Tizen.NUI.PropertyValue pValue = Tizen.NUI.Object.GetProperty(SwigCPtr, Renderer.Property.DepthIndex);
+                pValue?.Get(out temp);
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/Renderer.cs:51]: To keep the same logic as before, please remove `?`.
Tizen.NUI.Object.GetProperty is returning `new PropertyValue` which can not be null.

Comment (zhouleonlei) [reply src/Tizen.NUI/src/public/Renderer.cs]: > To keep same logic as before, please remove `?`.
> Tizen.NUI.Object.GetProperty is returning `new PropertyValue` which can not be null.

Fixed.

File: src/Tizen.NUI/src/public/ViewProperty/ShadowBase.cs [hunk 2]
```diff
@@ -209,8 +215,12 @@ private void SetTransformMap(PropertyMap transformMap)
                 return;
             }
 
-            transformMap.Find((int)VisualTransformPropertyType.Offset)?.Get(Offset);
-            transformMap.Find((int)VisualTransformPropertyType.ExtraSize)?.Get(Extents);
+            PropertyValue temp = transformMap.Find((int)VisualTransformPropertyType.Offset);
+            temp?.Get(Offset);
+            temp.Dispose();
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/ViewProperty/ShadowBase.cs:220]: PropertyMap.Find result can be null. Please add "?" or move them into if statement.

File: src/Tizen.NUI/src/public/ViewProperty/ShadowBase.cs [hunk 3]
```diff
@@ -209,8 +215,12 @@ private void SetTransformMap(PropertyMap transformMap)
...
+            temp = transformMap.Find((int)VisualTransformPropertyType.ExtraSize);
+            temp?.Get(Extents);
+            temp.Dispose();
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/ViewProperty/ShadowBase.cs:223]: Ditto.

File: src/Tizen.NUI/src/public/ViewProperty/ShadowBase.cs (hunk 2 above)
Comment (zhouleonlei) [reply src/Tizen.NUI/src/public/ViewProperty/ShadowBase.cs]: > PropertyMap.Find result can be null. Please add "?" or move them into if statement.

Fixed

File: src/Tizen.NUI/src/public/ViewProperty/ShadowBase.cs (hunk 3 above)
Comment (zhouleonlei) [reply src/Tizen.NUI/src/public/ViewProperty/ShadowBase.cs]: > Ditto.

fixed

**Review by rabbitfor — APPROVED** [review]
LGTM

**Review by AdunFang — APPROVED** [review]
It is OK.

**Review by AchoWang — APPROVED** [review]
Looks good to me.

**Pull request merged**